core/types/data_blob_tx.go

```diff
@@ -57,10 +56,6 @@ func (*ECDSASignature) FixedLength() uint64 {
 	return 1 + 32 + 32
 }
 
-func (sig *ECDSASignature) HashTreeRoot(hFn tree.HashFn) tree.Root {
```

Comment:
Is `HashTreeRoot` an interface? If we're deleting all the implementations on various structs, should we remove interface function definition as well?

Reply:
Yes it was implementing an interface but the interface is defined in the external ssz encoder library (protolambda/ztyp/tree).